crontabs/ldap

```diff
@@ -1 +1 @@
-* 5 * * * /entrypoint/backup
+* 5 * * * openldap /entrypoint/backup
```

Comment:
If the cron isn't run anymore, why is the crontab still around?

Reply:
This cron task is there to run /entrypoint/backup in order to dump the ldap database.
Since I re-based this image on debian, /entrypoint/backup is not run anymore because the cron task doesn't inherit env variables.
Those env variables contain ldap login && password